valid_service.md

```diff
@@ -0,0 +1,154 @@
+CONTAINER ID   IMAGE                                                   COMMAND                  CREATED       STATUS                   PORTS                                                                                  NAMES
+92721ba5430a   opea/gen-ai-comps:chatqna-ui-server                     "docker-entrypoint.s…"   5 hours ago   Up 10 minutes            0.0.0.0:5173->5173/tcp, :::5173->5173/tcp                                              chatqna-gaudi-ui-server
+a2df33f99975   opea/gen-ai-comps:chatqna-megaservice-server            "python chatqna.py"      5 hours ago   Up 5 hours               0.0.0.0:8888->8888/tcp, :::8888->8888/tcp                                              chatqna-gaudi-backend-server
+33bad778553b   opea/gen-ai-comps:embedding-tei-server                  "python embedding_te…"   5 hours ago   Up 5 hours               0.0.0.0:6000->6000/tcp, :::6000->6000/tcp                                              embedding-tei-server
+30f961b94b16   opea/gen-ai-comps:llm-tgi-gaudi-server                  "python llm_tgi.py"      5 hours ago   Up 5 hours               0.0.0.0:9000->9000/tcp, :::9000->9000/tcp                                              llm-tgi-gaudi-server
+fb9b6d84e7bd   opea/gen-ai-comps:reranking-tei-server                  "python reranking_te…"   5 hours ago   Up 5 hours               0.0.0.0:8000->8000/tcp, :::8000->8000/tcp                                              reranking-tei-gaudi-server
+115f81d80017   ghcr.io/huggingface/tgi-gaudi:1.2.1                     "text-generation-lau…"   5 hours ago   Exited (2) 5 hours ago                                                                                          tgi-gaudi-server
+7eb49377b256   ghcr.io/huggingface/text-embeddings-inference:cpu-1.2   "text-embeddings-rou…"   5 hours ago   Exited (2) 5 hours ago                                                                                          tei-xeon-server
+d277d25415fc   redis/redis-stack:7.2.0-v9                              "/entrypoint.sh"         5 hours ago   Up 5 hours               0.0.0.0:6379->6379/tcp, :::6379->6379/tcp, 0.0.0.0:8001->8001/tcp, :::8001->8001/tcp   redis-vector-db
+f89734740602   opea/gen-ai-comps:retriever-redis-server                "/home/user/comps/re…"   5 hours ago   Up 5 hours               0.0.0.0:7000->7000/tcp, :::7000->7000/tcp                                              retriever-redis-server
+```
+
+Unfortunately, here two containers are exited.
+
+```
+115f81d80017   ghcr.io/huggingface/tgi-gaudi:1.2.1                     "text-generation-lau…"   5 hours ago   Exited (2) 5 hours ago                                                                                          tgi-gaudi-server
+7eb49377b256   ghcr.io/huggingface/text-embeddings-inference:cpu-1.2   "text-embeddings-rou…"   5 hours ago   Exited (2) 5 hours ago
+```
+
+To get the container status, next check the container logs.
+
+## Check docker container log
+
+3. Check the log of container
```

Comment:
Same here.

Reply:
Also add another command to check the overall log:

`docker compose -f docker_compose.yaml logs`